Readme.md

```diff
@@ -5,22 +5,20 @@ based on [Ubuntu Cloud Images](https://cloud-images.ubuntu.com). It supports
 the `x86_64` and `aarch64` architectures.
 
 ### Setup
-
 Install factory and its dependencies (assumes Ubuntu 16.04):
 
 ```shell
-$ sudo apt install -y wget cloud-utils qemu-kvm genisoimage
-$ git clone https://github.com/liquidinvestigations/factory
-$ cd factory
-$ ./factory prepare-cloud-image
+$ sudo apt install -y wget qemu-kvm qemu-utils xz-utils
+$ python3 <(curl -sL https://github.com/liquidinvestigations/factory/raw/master/install.py) factory
+$ ./factory echo hello world
```

Comment:
Since the install script from the previous line installs stuff into `./factory`, shouldn't this be `./factory/factory echo ...`?

Reply:
Ah yes.